Australian Epic for end of year campaign

alexduf opened the pull request:
## What does this change?
This adds an ask at the bottom of the article for our Australian readers.
It will be displayed to 10% of the population, and will help us determine which variant performs best
One of the variants includes a cartoon from First Dog on the Moon. I have embedded it in our `static` folder and I'm using the `config.js` to reference it. Let me know if that approach causes any issue.

## What is the value of this and can you measure success?
We can determine what's the best way to ask for contributions and supporters in Australia, and will use the result of the test to do so.

## Does this affect other platforms - Amp, Apps, etc?
No

## Screenshots
![screenshot from 2016-12-14 14-39-48](https://cloud.githubusercontent.com/assets/3389563/21186300/f615db76-c20b-11e6-8641-ca8ba924fec1.png)
![screenshot from 2016-12-14 14-39-21](https://cloud.githubusercontent.com/assets/3389563/21186302/f6161cd0-c20b-11e6-8b8f-8554078e31f5.png)
![screenshot from 2016-12-14 14-38-58](https://cloud.githubusercontent.com/assets/3389563/21186301/f615e7f6-c20b-11e6-9af2-02c939af4878.png)
![screenshot from 2016-12-14 14-38-27](https://cloud.githubusercontent.com/assets/3389563/21186303/f6174b32-c20b-11e6-970f-7669cfce155b.png)

cc @guardian/guardian-frontend @desbo

jranks123 commented on static/src/javascripts/projects/common/modules/experiments/tests/contributions-epic-on-the-moon.js:
```diff
@@ -0,0 +1,234 @@
+            mediator.on(epicViewedEvent, complete);
+        }
+
+        var canBeDisplayed = function() {
```
This function is not necessary, all the logic around whether the test can run is in the canRun function on line 54.

alexduf replied:
Good point! I'm fixing it.

alexduf commented:
I had another go at it, would you like to check again @jranks123 ?

regiskuckaertz commented:
Could @paperboyo work his magic and decrease the size of these images?

jranks123 commented:
Looks good from my end 👍

paperboyo commented:
Yes, Sir! Bit lossy but should be OK.

![first-dog-mb_opt2](https://cloud.githubusercontent.com/assets/6032869/21221549/ae11a5ee-c2b5-11e6-8634-58bbd5c0bb42.png)
![first-dog-dt_opt3](https://cloud.githubusercontent.com/assets/6032869/21221550/ae12b024-c2b5-11e6-87a4-b6594ef9ab7f.png)

alexduf commented:
Here's what I have done:

- Rebased to get the latest test flags so the build can pass
- Added logic to avoid test clash
- fix @jranks123 comments
- replaced images for lighter ones from @paperboyo 

Let me know if there's anything else I should fix

jranks123 commented:
👍
